src/types/FilterTodosBy/index.ts

```diff
@@ -0,0 +1 @@
+export * from './FilterTodoBy';
```

Comment:
Avoid exporting directly from an import. Use a clear way to import and a clear way to export to maintain consistency.